chapters/depth.md

```diff
@@ -0,0 +1,260 @@
+# Depth
+
+The term `depth` is used in various spots in the [Vulkan Spec](./vulkan_spec.md). This chapter is aimed to give an overview of the various "depth" terminology used in Vulkan. Some basic knowledge of 3D graphics is needed to get the most out of this chapter.
+
+> Note: While stencil is closely related depth, this chapter does not aim to cover it outside the realm of API names
+
+- [Graphics Pipeline](#graphics-pipeline)
+- [Formats](#formats)
+- [Depth Buffer as a VkImage](#depth-buffer-as-a-vkimage)
+  * [Layout](#layout)
+  * [Clearing](#clearing)
+- [Pre-rasterization](#pre-rasterization)
+  * [Primitive Clipping](#primitive-clipping)
+    + [User defined clipping and culling](#user-defined-clipping-and-culling)
+    + [Porting from OpenGL](#porting-from-opengl)
+  * [Viewport Transformation](#viewport-transformation)
+    + [Depth Range](#depth-range)
+- [Rasterization](#rasterization)
+  * [Depth Bias](#depth-bias)
+- [Post-rasterization](#post-rasterization)
+  * [Fragment Shader](#fragment-shader)
+    + [Conservative depth](#conservative-depth)
+  * [Per-sample processing and coverage mask](#per-sample-processing-and-coverage-mask)
+    + [Resolving depth buffer](#resolving-depth-buffer)
+  * [Depth Bounds](#depth-bounds)
+  * [Depth Test](#depth-test)
+    + [Depth Compare Operation](#depth-compare-operation)
+    + [Depth Buffer Writes](#depth-buffer-writes)
+    + [Depth Clamping](#depth-clamping)
+
+# Graphics Pipeline
+
+The concept of "depth" is only used for [graphics pipelines](./what_vulkan_can_do.md) in Vulkan and doesn't take effect until a draw call is submitted.
```

Comment:
It seems strange to say a "concept" "takes effect". It's also not really true that depth doesn't apply before draw calls - the depth buffer can be cleared for example.

Maybe something like:

```suggestion
The concept of "depth" is only used for [graphics pipelines](./what_vulkan_can_do.md) in Vulkan. Depth operations are only take place after a draw call is submitted.
```